### ISSUE TITLE
PEP 3105 Make print a function in docs/names and docs/pair

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8413]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8413|
|Type|enhancement|
|Created|2016-06-03 21:21:18Z|
|Branch|https://github.com/twisted/twisted/tree/8413-rodrigc-print|

This ticket was split out of [ticket:5812] which was too big to review.

PEP 3105 ( ​​​https://www.python.org/dev/peps/pep-3105/ ) **print** deprecates has changed from a statement to a function.

In Python 2.6 and Python 2.7, the default is print as a statement

```
print "Hello"
```

In Python 3.x, the above example is an error, since only print as a function is available.

To consistently use **print** as a function
on Python 2.6, Python 2.7, and Python 3.x, this should be done:

```
from __future__ import print_function

print("Hello")
```

<details><summary>Searchable metadata</summary>

```
trac-id__8413 8413
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8413_rodrigc_print 8413-rodrigc-print
branch_author__ 
status__closed closed
resolution__fixed fixed
component__names names
keywords__None None
time__1464988878243576 1464988878243576
changetime__1465982523165724 1465982523165724
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/129

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Look good. thanks!

Will merge.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset bc8d3b18f18c103e51709429aa1b84c30d133afb
```
#!CommitTicketReference repository="" revision="bc8d3b18f18c103e51709429aa1b84c30d133afb"
Merge 8413-rodrigc-print: Use print function in docs/names and docs/pair.

Author: rodrigc
Reviewer: adiroiban
Fixes: #8413
```
